src/paperqa/clients/__init__.py

```diff
@@ -197,16 +197,16 @@ async def bulk_query(
         )
 
     async def upgrade_doc_to_doc_details(self, doc: Doc, **kwargs) -> DocDetails:
-
-        # note we have some extra fields which may have come from reading the doc text,
-        # but aren't in the doc object, we add them here too.
-        extra_fields = {
```

Comment:
The usage of "extra" here is an overloaded term:

- Extra in a Pydantic context (and we have `DocDetails.other`, to add to the confusion)
- Extra in a query context

Paired with vagueness such as "some extra fields", I decided to rename and reworded this whole section